pages/success-banner/index.js

```diff
@@ -1,13 +1,47 @@
 import React from 'react';
 import Head from 'next/head';
 
-class Button extends React.Component {
+const eventLog = (eventDesc) => {
+  const data = 'Banner ' + eventDesc;
+
+  return () => {
+    const url = new URL(location.href);
+
+    if (url.searchParams.get('etcher-version').indexOf('1.0.0') !== -1) {
```

Comment:
If we want to check these kinds of things, we should make Etcher expose a "web banner" version number instead of using Etcher's main version. E.g. we can say "web banner messaging protocol version 1", and then independently increment that when we need to. If such protocol version property is unset, then we infer is the oldest one.